queens-psyc-clinic/app
# FE Admin populated cards

### Admin Populated Card Component
This is the admin populated cards component done!

### Testing
1. navigate to branch: git checkout `admin-cards`
2. cd to the public folder
3. run `npm install`
4. run `npm start`
5. ensure the cards are showing, no errors, and feel free to checkout the mock data! (Also, made it responsive so it should control overflow when the screen size changes 👀)

<img width="500" alt="Screenshot 2024-03-01 at 11 56 33 AM" src="https://github.com/queens-psyc-clinic/app/assets/74020381/f3c3b2e8-d44b-4ca8-ba62-ab1945ae9f5b">

For reference checkout: #38
